src/components/Tool/Appeal/Shared/useGetExcludedReasons/useGetExcludedReasons.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { getLocalizedExcludedFromAppealReasons } from 'src/utils/functions/getLocalizedExcludedFromAppealReasons';
+import { ExcludedAppealContactInfoFragment } from '../AppealExcludedContacts.generated';
+
+// The return value doesn't change until `delay` milliseconds have elapsed since the last time `value` changed
+export const useGetExcludedReasons = (
+  excludedContacts: ExcludedAppealContactInfoFragment[],
+  contactID: string,
+): string[] => {
+  const { t } = useTranslation();
+
+  const reasons = useMemo(() => {
+    if (!excludedContacts.length || !contactID) {
```

Comment:
Is the `!contactID` check necessary? I don't think contacts can have an empty string id.

Reply:
Actually, I think the entire check can be removed. If `excludedContacts` is empty, `find` won't find anything, and it will bail return on line 21.